requirements.txt

```diff
@@ -6,9 +6,11 @@ six>=1.11.0
 h5py>=2.10.0
 numpy>=1.15.4
 pandas>=0.23.4
+sklearn>=0.0
```

Comment:
I believe the proper package is actually `scikit-learn`, based on the `sklearn` package info https://pypi.org/project/sklearn/

Reply:
Done